cadquery/occ_impl/shapes.py

```diff
@@ -2091,6 +2092,26 @@ def makeLine(cls, v1: VectorLike, v2: VectorLike) -> "Edge":
             BRepBuilderAPI_MakeEdge(Vector(v1).toPnt(), Vector(v2).toPnt()).Edge()
         )
 
+    @classmethod
+    def makeBezier(cls, points: List[Vector]) -> "Edge":
+        """
+        Create a cubic Bézier Curve from the points.
+
+        :param listOfVector: a list of Vectors that represent the points.
```

Comment:
Docstring param name mismatch. 
The docs list both points, and listOfVector:
https://cadquery--1529.org.readthedocs.build/en/1529/classreference.html

Reply:
Fixed! Thanks!